srcCpp/PerftestTransport.cxx

```diff
@@ -712,6 +730,14 @@ void PerftestTransport::printTransportConfigurationSummary()
         stringStream << "\tNic: " << allowInterfaces << "\n";
     }
 
+    stringStream << "\tUse Multicast: " << ((allowsMulticast())? "True" : "False");
+    if(!allowsMulticast() && useMulticast){
```

Comment:
Can you add a space in the next places:
 ){
if(
!allowsMulticast

Reply:
DONE